src/pages/home/ReportScreen.js

```diff
@@ -160,12 +156,29 @@ function ReportScreen({
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const {isSmallScreenWidth} = useWindowDimensions();
-
-    const firstRenderRef = useRef(true);
+    const {isOffline} = useNetwork();
     const flatListRef = useRef();
     const reactionListRef = useRef();
+    const firstRenderRef = useRef(true);
+    const reportIDFromRoute = getReportID(route);
```

Comment:
Try and look through this file to replace more usages of `getReportID(route)` with `reportIDFromRoute`

Reply:
👍🏻